File: app/src/main/java/org/ole/planet/myplanet/callback/OnHomeItemClickListener.java
package org.ole.planet.myplanet.callback;

import androidx.fragment.app.Fragment;

import org.ole.planet.myplanet.model.RealmMyLibrary;
import org.ole.planet.myplanet.model.RealmStepExam;

public interface OnHomeItemClickListener {
    void openCallFragment(Fragment f);

    void openLibraryDetailFragment(RealmMyLibrary library);

    void showRatingDialog(String resource, String resource_id, String title, OnRatingChangeListener listener);

    void sendSurvey(RealmStepExam current);
}

File: app/src/main/java/org/ole/planet/myplanet/callback/OnHomeItemClickListener.kt
package org.ole.planet.myplanet.callback

import androidx.fragment.app.Fragment
import org.ole.planet.myplanet.model.MyLibrary
import org.ole.planet.myplanet.model.StepExam

interface OnHomeItemClickListener {
    fun openMyFragment(f: Fragment)
    fun openCallFragment(f: Fragment)
    fun openLibraryDetailFragment(library: MyLibrary?)
    fun showRatingDialog(type: String?, resourceId: String?, title: String?, listener: OnRatingChangeListener?)

    fun sendSurvey(current: StepExam?)
}
